jsparagus/gen.py

```diff
@@ -2682,219 +2705,216 @@ def visit(aps):
         self.aps_visitor(self.aps_start(state), visit)
         return record
 
-    def fix_with_context(self, s, aps_lanes):
-        raise ValueError("fix_with_context: Not Implemented")
```

Comment:
This function is still called by the test suite. Except that instead of a Not implemented here, we would have a missing method error.

Reply:
Hmm. The test suite didn't complain. I will uncomment these lines, though.